lighthouse-core/report/html/templates.html

```diff
@@ -247,7 +247,7 @@
     .lh-export {
       position: absolute;
       right: var(--section-indent);
-      transform: translateY(0);
```

Comment:
mind a brief explanation of why we're doing this instead of increased z-index? :)

Reply:
thanks will do.